templates/csv.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
           }
         },
         {
-          "apiVersion": "operator.knative.dev/v1alpha1",
+          "apiVersion": "operator.knative.dev/v1beta1",
```

Comment:
perhaps we could move the `knative kafka` bits to v1beta1 as well - but in a separate PR?

Reply:
Absolutely I thought about this too so we dont give the false impression that Kafka is left behind.